docs/DeveloperGuide.adoc

```diff
@@ -775,39 +885,90 @@ Use case resumes at step 2.
 .  Should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-{More to be added}
+.  A user should be able to search for and locate a contact within 5 seconds.
+.  Contact list data should not be corrupted even in the event of the application being forcefully terminated while processing.
+.  A user should not be able to accidentally clear their contact list.
+.  A user's contact list data should not be viewable to anyone else who is not authorised.
```

Comment:
please look at the project constraints. this one doesn't seem consistent with the given constraints